docs/guide/tutorials/dask-cluster.md

```diff
@@ -0,0 +1,107 @@
+
+```python
+from prefect import task, Flow
+import datetime
+import random
+from time import sleep
+
+
+@task
+def inc(x):
+    sleep(random.random() / 10)
+    return x + 1
+
+
+@task
+def dec(x):
+    sleep(random.random() / 10)
+    return x - 1
+
+
+@task
+def add(x, y):
+    sleep(random.random() / 10)
+    return x + y
+
+
+@task(name="sum")
+def list_sum(arr):
+    return sum(arr)
+
+
+with Flow("dask-example") as flow:
+    incs = inc.map(x=range(100))
+    decs = dec.map(x=range(100))
+    adds = add.map(x=incs, y=decs)
+    total = list_sum(adds)
+```
+
+So far, all we have done is define a flow which contains all the necessary information for how to run these tasks - none of our custom task code has been executed yet. 
+To have this flow run on our Dask cluster, all we need to do is provide an appropriately configured `DaskExecutor` to the `flow.run()` method:
+
+```python
+from prefect.engine.executors import DaskExecutor
+
+executor = DaskExecutor(address="tcp://10.0.0.41:8786")
+flow.run(executor=executor)
+```
+
+If you happen to have `bokeh` installed, you can visit the [Dask Web UI](https://distributed.readthedocs.io/en/latest/web.html) and see your tasks being processed when the flow run begins!
+
+## Next Steps
+Let's take this one step further: let's attach a schedule to this flow, and package it up so that we can point it to any Dask cluster we choose, without editing the code which defines the flow. To do this, we will first add a main method to our script above so that it can be executed via CLI:
+
+```python
+def main():
+    from prefect.schedules import IntervalSchedule
+
+    every_minute = IntervalSchedule(start_date=datetime.datetime.utcnow(),     
+                 					interval=datetime.timedelta(minutes=1))
+    flow.schedule = every_minute
+    flow.run() # runs this flow on its schedule
+
+
+if __name__ == "__main__":
+    main()
+```
+
+Notice that we didn't specify an executor in our call to `flow.run()`. This is because the default executor can be set via environment variable (for more information on how this works, see [Prefect's documentation](../core_concepts/configuration.html)). Supposing we save this in a file called `dask_flow.py`, we can now specify the executor and the Dask scheduler address as follows:
+
+```bash
+> export PREFECT__ENGINE__EXECUTOR__DEFAULT_CLASS="prefect.engine.executors.DaskExecutor"
+> export PREFECT__ENGINE__EXECUTOR__DASK__ADDRESS="tcp://10.0.0.41:8786"
+
+> python dask_flow.py
+```
+
+This flow will now run every minute on your local Dask cluster until you kill this process.
+
+## Further steps
+Take this example to the next level by storing this flow in a Docker container and deploying it with Dask on Kubernetes using the excellent [dask-kubernetes](http://kubernetes.dask.org/en/latest/) project! Details are left as an exercise to the reader.
```

Comment:
This line feels a little too much like homework ;)
```suggestion
Take this example to the next level by storing your flow in a Docker container and deploying it with Dask on Kubernetes using the excellent [dask-kubernetes](http://kubernetes.dask.org/en/latest/) project!
```

Reply:
Hahah this is a fair point, but I included this as a joke for all the math people out there... maybe if I include a winky face that will be more apparent?

Reply:
I thought that might be it, haha - go for it :)

Reply:
Updated - I changed the wording from `this` to `your` per your suggestion as well.  One day you'll need to teach me how to properly use `which` and `that` haha

Reply:
Ha - I don't know why this is the *one* grammar rule I can't unsee (because I'm far from perfect elsewhere). As it was explained to me:
- "which" introduces a phrase that isn't important to the sentence, but just provides further detail. A "which" phrase can usually be offset with commas.
- "that" introduces a phrase that is critical to the sentence

`The car that won the race is fast.`

A specific car, implied to be one of many, whose specific identity is indicated by the `that` phrase.

`The car, which won the race, is fast.`

A single car, with no implication of other cars.